test/WebSites/RazorCompilerCacheWebSite/Startup.cs

```diff
@@ -9,17 +9,19 @@ namespace RazorCompilerCacheWebSite
 {
     public class Startup
     {
+        // Set up application services
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add MVC services to the services container
```

Comment:
nit: this comment is getting a little old by this point.  says nothing `services.AddMvc()` doesn't